assets/blueprints/radiswap/src/lib.rs

```diff
@@ -1,195 +1,124 @@
+use scrypto::blueprints::pool::*;
 use scrypto::prelude::*;
 
+// =================================================================================================
+// TODO: The interface of this can be made better once we have a way to generate stubs for native
```

Comment:
Got a plan on how to address this for pool and other native blueprints. No need to focus too much on this in this PR.